framework/helpers/BaseConsole.php

```diff
@@ -1019,4 +1019,59 @@ public static function endProgress($remove = false, $keepPrefix = true)
         self::$_progressEtaLastDone = 0;
         self::$_progressEtaLastUpdate = null;
     }
+
+    public static function table($headers, $rows, $span = '+', $separator = '-', $conector = '|')
```

Comment:
do you suggest include this options : 

```
'top': '═' , 'top-mid': '╤' , 'top-left': '╔' , 'top-right': '╗'
         , 'bottom': '═' , 'bottom-mid': '╧' , 'bottom-left': '╚' , 'bottom-right': '╝'
         , 'left': '║' , 'left-mid': '╟' , 'mid': '─' , 'mid-mid': '┼'
         , 'right': '║' , 'right-mid': '╢' , 'middle': '│'
```

For example : 

```
public function table(array $headers,array $rows,array $chars);
```
Where $chars is array with the options 
described before.